docs/guides/getting-started.md

```diff
@@ -19,24 +19,30 @@ You can request some via the [L14 faucet](http://faucet.l14.lukso.network/).
   <TabItem value="web3js" label="web3.js" default>
 
 ```javascript
-const Web3 = require("web3");
+const Web3 = require('web3');
```

Comment:
Ah I see `require` is used in a few places in this standards section. If we can switch them to import statements I think  it would be cool as `require` loses types when importing packages